src/content/structured/get-started/test-components.mdx

```diff
@@ -58,6 +58,5 @@ Therefore, testing components within the shadow DOM can be problematic because e
 
 To ease the process of testing, here are guides to testing our UI Kit components in different testing frameworks.
 
-- [Jest](/get-started/test-components/testing-with-jest)
-- [React Testing Library](/get-started/test-components/testing-with-react-testing-library)
+- [Jest and RTL](/get-started/test-components/testing-with-jest-and-rtl)
```

Comment:
RTL hasn't been defined on this page so I'd write it out in full in the link text.